### ISSUE TITLE
[ENH] Enable Exchange of skobjects without Multiplexer in graphical pipelines

### ISSUE THREAD

**Issue by benHeid** [sktime/sktime#5217]
**Is your feature request related to a problem? Please describe.**
Currently, the graphical pipeline requires a Multiplexer to exchange for example forecasters during a grid search. However, this nesting could be avoided.

**Describe the solution you'd like**
In a grid search enable the following key `<step_name>_skobject`. As values, multiple forecasters could be provided. This solution should be easy to implement. It would just affect the set_params methods of the graphical pipeline and should look similar as setting the new edges. 

**Describe alternatives you've considered**
Stick with the Multiplexers

**Comment by fkiraly** [comment]
good idea - using the existing syntax.

As a small comment on this: there are actually two important "parallel" operations - the "feature union" and the "multiplexer". In typical use cases, feature union is more frequently used than the multiplexer.

I do not have a clear preference, I would just advise whether the proposed syntactic space is better given to feature union rather than the multiplexer? Or is there already intuitive syntax for a feature union and this would be duplicating it?

**Comment by benHeid** [comment]
Yes it the graphical pipeline using a list when passing arguments to x or y in the edges parameter is equivalent to the FeatureUnion.

**Comment by 0149Sailesh** [comment]
Hi @fkiraly! I would love to work on this issue if it is still open and relevant.

**Comment by urvashi-agrawal-dev** [comment]
> **Is your feature request related to a problem? Please describe.** Currently, the graphical pipeline requires a Multiplexer to exchange for example forecasters during a grid search. However, this nesting could be avoided.
> 
> **Describe the solution you'd like** In a grid search enable the following key `<step_name>_skobject`. As values, multiple forecasters could be provided. This solution should be easy to implement. It would just affect the set_params methods of the graphical pipeline and should look similar as setting the new edges.
> 
> **Describe alternatives you've considered** Stick with the Multiplexers

can i start working on this issue ?can u assign it to me ??
